src/logger/src/logger.rs

```diff
@@ -268,7 +263,7 @@ impl Logger {
     ///
     /// fn main() {
     ///     let l = LOGGER.deref();
-    ///     l.set_level(log::Level::Info);
+    ///     l.set_level(log::LevelFilter::Info);
```

Comment:
I think the documentation also needs to be updated. Is the default level still warning?

Reply:
The default is Warning only in the vmm crate. The logger crate does not have a default any more and this is the documentation from the logger crate. Am I missing something?

Reply:
@serban300 I think @andreeaflorescu  is referring to these 2 comments:  https://github.com/firecracker-microvm/firecracker/blob/4687bc8a9a8e23cdd6c258c656a23e665be0c49a/src/logger/src/logger.rs#L147 and https://github.com/firecracker-microvm/firecracker/blob/4687bc8a9a8e23cdd6c258c656a23e665be0c49a/src/logger/src/logger.rs#L254. We need to carve out the part that mentions a default level as from now on the default level will be that of the underlying log crate.

Reply:
You're right. Done.

Reply:
Actually let's keep the default level in order to make sure we keep exactly the same behavior as before.